src/ui/WebView.cpp

```diff
@@ -134,7 +142,7 @@ namespace wfl::ui
                 webView->signalNotification().emit(true);
             }
 
-            g_signal_connect(notification, "clicked", G_CALLBACK(notificationDestroyed), webView);
```

Comment:
I remember that this line is intended to make tray icon normal again when the notification is clicked. Please check that if notifying `closed` covers clicking as well by checking whether the tray icon goes back to normal state when you click on a notification, otherwise keep this line.

Reply:
Yes. I can confirm that when the notification arrives, the icon gets a red dot; and when the notification is clicked, the red dot goes away.

Even previously, when a notification arrived and the window was manually opened by the user without interaction with the notification, the red dot would go away. I suppose that is what takes care of this issue you brought to light.